src/main/resources/application.yml

```diff
@@ -70,8 +70,7 @@ nakadi:
     delivery.timeout.ms: 30000 # request.timeout.ms + linger.ms
     max.block.ms: 5000 # kafka default 60000
   zookeeper:
-    kafkaNamespace:
-    brokers: zookeeper:2181
+    connection_string: zookeeper://zookeeper:2181
```

Comment:
This is the first variable with underscore in this file.

Reply:
Will use camel case